src/clr-angular/data/datagrid/datagrid.spec.ts

```diff
@@ -300,7 +300,7 @@ export default function(): void {
 
       describe('Template API', function() {
         it('sets the currentSingle binding', function() {
-          expect(selection.currentSingle).toBeNull();
+          expect(selection.currentSingle).toBeUndefined();
```

Comment:
I don't see any changes to the `currentSingle` declaration. How did this test pass before?